### DISCUSSION TITLE
Closed Caption for the hearing impaired

### DISCUSSION THREAD

**Discussion by JamesRabe** [EDCD/EDDI#2519] [Needs thought]
How do I turn on closed captioning?  I am unable to find the setting, so I assume it hasn't been included yet?  Just in case it has I thought I'd ask how before I requested the feature.

**Comment by Tkael** [comment]
EDDI does not currently support closed captioning or any other overlay. That said, if you have software that can display lines of text from a text file then you can potentially generate an overlay from the contents of `speechresponder.out` (which can optionally be activated from the Speech Responder tab and will write to a file at %appdata%/EDDI).

**Reply by JamesRabe** [reply]
I think, if it works as described, that this will be even better than CC.  Thanks.

EDIT: Been trying it for a few weeks and it does work better than a CC overlay, at least for me.  It's more work, but it organizes the information better.
